src/main/resources/react/src/pages/AddGoods/AddGoods.js

```diff
@@ -146,212 +146,143 @@ const AddGoods = () => {
-                  <CheckBox
-                    text="0"
-                    margin="0 0 15px 0"
-                    fontSize="14px"
-                    checked={false}
-                    click={null}
-                  />
-                  <CheckBox
-                    text="1-2"
-                    margin="0 0 15px 0"
-                    fontSize="14px"
-                    checked={false}
-                    click={null}
-                  />
-                  <CheckBox
-                    text="2-4"
-                    margin="0 0 15px 0"
-                    fontSize="14px"
-                    checked={false}
-                    click={null}
-                  />
-                  <CheckBox
-                    text="5-7"
-                    margin="0 0 15px 0"
-                    fontSize="14px"
-                    checked={false}
-                    click={null}
-                  />
-                  <CheckBox
-                    text="8-11"
-                    margin="0 0 15px 0"
-                    fontSize="14px"
-                    checked={false}
-                    click={null}
-                  />
-                  <CheckBox
-                    text="11-14"
-                    margin="0 0 15px 0"
-                    fontSize="14px"
-                    checked={false}
-                    click={null}
-                  />
+              <div className="characteristics">
+                <h3>{getTranslatedText(`addAdv.options`, lang)}</h3>
+                <div className="characteristics_items">
+                  <div className="characteristics_item">
+                    <h4>{getTranslatedText(`addAdv.age`, lang)}:</h4>
+                    {ages.map((item, idx) => (
+                      <FormikCheckBox
+                        key={item + idx}
+                        text={enumAge[item]}
+                        value={item}
+                        name="age"
+                        type="checkbox"
+                        margin="0 0 15px -7px"
+                      />
+                    ))}
+                  </div>
+                  <div className="characteristics_item">
+                    <h4>{getTranslatedText(`addAdv.sex`, lang)}:</h4>
+                    {sex.map((item, idx) => (
+                      <FormikCheckBox
+                        key={item + idx}
+                        text={getTranslatedText(`genderEnum.${item}`, lang)}
+                        value={item}
+                        name="gender"
+                        type="radio"
+                        margin="0 0 15px -7px"
+                      />
+                    ))}
+                  </div>
+                  <div className="characteristics_item">
+                    <h4>{getTranslatedText(`addAdv.size`, lang)}:</h4>
+                    {size.map((item, idx) => (
+                      <FormikCheckBox
+                        key={item + idx}
+                        text={`${item} см`}
```

Comment:
If I select English locale what will be displayed for the size mark?

Reply:
I will translate in the future

Reply:
Got it.